lib/matplotlib/tests/test_mathtext.py

```diff
@@ -110,6 +110,9 @@
     r'$\left(X\right)_{a}^{b}$',  # github issue 7615
     r'$\dfrac{\$100.00}{y}$',  # github issue #1888
 ]
+lightweight_math_tests = [
```

Comment:
Please add a comment in what sense they are "lightweight", i.e. how this works.

Reply:
done